src/book/02-system/03-tools/06-natnet.mdx

```diff
@@ -0,0 +1,104 @@
+
+4. Make sure there are no motion capture dots on anything on the soccer field.
+
+5. On the left hand side of the screen, you will see a Camera Calibration panel, and up the top you will see the button "Mask visible". Click this. This will cover up any static reflective objects in the cameras' views (including the active infra-red emissions from other cameras). 
+
+6. Once all the bright patches have been masked with red, click on the "Start Wanding" button next to the button clicked in step 4. Use the wanding stick with three MoCap dots on the cross bar, and slowly make your way around the volume you wish to work in, waving the wand around. You will be able to see coloured lines in the camera boxes. You want to aim to fill as much of each camera box as possible to achieve the best calibration. Each camera has a ring of green LEDs- as you start to wave the wand in front of each camera, each ring will slowly fill up. Aim to completely fill up each camera’s ring.
+
+7. After you have finished wanding, click the "calibrate" button further down in the side panel.
+
+8. The side panel will now switch to the ground plane tab. Find the Ground Plane L Frame (It is black with 3 MoCap dots on it and two bubble levels), and place in on the ground at the center circle (normally with the Z axis pointing towards the goals with the brick wall behind it. Don't bother trying to level out the L frame as it seems to be a hopeless task). Back on the computer, click the "Set Ground Plane" button. This will produce a pop-up window prompting you to save your calibration. Click save. Congratulations, your calibration is complete.
+
+
+</details>
+
+1. Follow the [Getting Started instructions](/guides/main/getting-started).
+
+2. Start the Motive software, it doesn't matter if motive is running locally or on another machine. Ensure that the [Data Streaming option](https://docs.optitrack.com/motive/data-streaming) in Motive is enabled. This will start sending packets to the Motive defined default multicast address on the local network.
+
+3. Once you have the NUbots codebase running on your OS of choice and Motive running and Streaming packets to the network, NatNet can be run in one of two ways:
+
+<details>
+<summary>On the Robot</summary>
+
+This is the default method for running NatNet. If you wish to output motive packets directly to file for debugging purposes, change the dump_packets option in NatNet.yaml to true. 
+
+1. Ensure that the NatNet role has been installed on the robot.
+
+2. [SSH into the robot](guides/main/getting-started#running-on-a-real-robot).
+
+3. `cd config`
+
+4. `nano DataLogging.yaml`
+
+5. Scroll down to message.input.MotionCapture and set to true. Additionally, set to true any of the sub messages that you want to use such as ".RigidBody". If you want "output.CompressedImage" or "input.sensors" logged, you can set them to true as well. Don't log "input.Image" as this will consume huge amounts of storage.
+
```

Comment:
have you been working off an old branch? very few messages are in the DataLogging yaml file, the user will need to add in `message.input.MotionCapture`. `message.input.Image` shouldn't be there either.

Reply:
I pretty much took this stuff straight from the original setup document that we used to set up mocap so I wasn't sure what had changed